shuup/admin/modules/orders/json_order_creator.py

```diff
@@ -51,6 +51,11 @@ def safe_get_first(model, **lookup):
         # A little helper function to clean up the code below.
         return model.objects.filter(**lookup).first()
 
+    @staticmethod
+    def is_empty_address(address_data):
```

Comment:
You should also probably check the other changes in this file and around it too.